cabal-testsuite/PackageTests/WithRepl/cabal.with-repl-invalid-path.out

```diff
@@ -0,0 +1,10 @@
+# cabal clean
+# cabal v2-repl
+Resolving dependencies...
+Build profile: -w ghc-<GHCVER> -O1
+In order, the following will be built:
+ - cabal-with-repl-0.1.0.0 (interactive) (lib) (first run)
+Configuring library for cabal-with-repl-0.1.0.0...
+Preprocessing library for cabal-with-repl-0.1.0.0...
+Error: [Cabal-7125]
+repl failed for cabal-with-repl-0.1.0.0-inplace.
```

Comment:
I feel like this should provide more information about the failure somewhere.

Reply:
The error you get is

```
foo: createProcess: posix_spawnp: does not exist (No such file or directory)
```